widgetry/src/widgets/button.rs

```diff
@@ -220,7 +276,14 @@ pub enum BtnBuilder {
         unselected_bg_color: Color,
         selected_bg_color: Color,
     },
-    Custom(GeomBatch, GeomBatch, Polygon, Option<Text>),
+    Custom {
+        normal: GeomBatch,
+        hovered: GeomBatch,
+        hitbox: Polygon,
+        maybe_tooltip: Option<Text>,
+        // thickness, color
+        maybe_outline: Option<(f64, Color)>,
```

Comment:
Alternatively, I could introduce a new BtnBuilder::ButtonWithOutline or something and leave BtnBuilder::Custom as it was, but this approach seemed least worst.

Reply:
LGTM. Now that we have some more explicit design guidance on button styles, I'm hoping to reduce the number of variations here and rethink this crazy enum soon.